tsconfig.base.json

```diff
@@ -1,17 +1,19 @@
 {
   "compilerOptions": {
-    "allowJs": true,
     "baseUrl": "./src",
     "declaration": true,
     "declarationMap": true,
     "esModuleInterop": true,
+    "isolatedDeclarations": true,
+    "isolatedModules": true,
     "lib": ["ES2023"],
     "module": "ESNext",
     "moduleResolution": "Bundler",
     "noImplicitAny": true,
     "outDir": "./dist",
     "skipLibCheck": true,
     "sourceMap": true,
-    "target": "ES2020"
+    "target": "ES2020",
```

Comment:
should we bump this to ES2024?

Reply:
sure!

Reply:
https://github.com/readmeio/oas/pull/933/commits/a5ad9e1e07fbd0e98e3068b8a46500e0d352e376